aws/artifacts/outputs.tf

```diff
@@ -0,0 +1,13 @@
+output "access_key_id" {
+  value       = "${module.user.access_key_id}"
+  description = "The access key ID for the user with permission to read/write to this bucket."
+}
+
+output "secret_access_key" {
+  value       = "${module.user.secret_access_key}"
+  description = "The secret access key. This will be written to the state file in plain-text."
+}
+
+output "bucket_name" {
+  value = "${module.bucket.bucket_id}"
```

Comment:
```suggestion
  value       = "${join("", module.bucket.*.bucket_id)}"
  description = "The name of the bucket"
```

Reply:
Don't need to use this pattern with modules.